cloudinit/net/cmdline.py

```diff
@@ -218,13 +246,28 @@ def config_from_klibc_net_cfg(files=None, mac_addrs=None):
     return {'config': entries, 'version': 1}
 
 
+def _get_netplan_net_cfg_files():
+    return glob.glob('/run/netplan/*.yaml')
```

Comment:
This needs to be sorted to ensure we have consistent config in the case there is more than one file.